apps/browser/src/autofill/notification/bar.ts

```diff
@@ -146,8 +208,12 @@ async function initNotificationBar(message: NotificationBarWindowMessage) {
   } = notificationBarIframeInitData;
   const i18n = getI18n();
   const resolvedTheme = getResolvedTheme(theme ?? ThemeTypes.Light);
+  const resolvedType = resolveNotificationType(notificationBarIframeInitData);
 
   if (useComponentBar) {
```

Comment:
non-blocking nit - no need to resolve for v2

```suggestion

  if (useComponentBar) {
    const resolvedType = resolveNotificationType(notificationBarIframeInitData);
```